lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * dummylayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
```

Comment:
See all other files and Browserify.
Summary:
Don't declare globals. Do this:
[ ] declare a local variable like
`var buildGraph = (...)…`
[ ] on the last line of your file, indicate to Browserify that that function you made (or object, or class, etc…) is the thing that this file should provide when you require it.
`module.exports = buildGraph`